assets/_scss/publication.scss

```diff
@@ -0,0 +1,103 @@
+/* Publication page customizations */
+
+.publication-cover {
+  padding-top: 0;
+  color: white;
+
+  .cover-img {
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+    // Enforce A4 aspect ratio.
+    height: 0;
+    padding-bottom: 141%;
+  }
+
+  .titles {
+    // A heuristic to make a bit more space for titles on mobile.
+    height: 100px;
+
+    .publication-type {
+      text-transform: uppercase;
+      font-weight: bold;
+      padding-block: 1rem;
+    }
+  }
+
+  .sponsor {
+    .title { font-weight: bold; }
+    a { color: white; }
+    .disclaimer {
+      margin-top: 0.5rem;
+      font-size: 0.9rem;
+      color: $c-lighter-grey;
+    }
+  }
+}
+
+.download {
+  margin-block: 2rem;
+  a { font-size: 1.4rem; }
+}
+
+.longread {
```

Comment:
Do we want this styling (especially the headings and arrow lists) for all our publication pages? It seems a bit too opinionated to my liking.

Reply:
This is an outdated version of the file: in the latest one, I have this guarded by a document class.